lib/querystring.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  StorageObject,
+  percentEscape,
+  unescapeBuffer
+} = require('internal/querystring');
 const QueryString = module.exports = {
   unescapeBuffer,
```

Comment:
Off-topic: `unescapeBuffer`, `encode` and `decode` are all removed from the docs, mind adding a comment?
(Not sure about which stage of deprecation they are in)